external-crates/move/crates/move-decompiler/tests/bytecode/output/370092bfce3c986971f00a4f54bc2a6cc98dfcdacc48cd48626c6bc66d685f71/cat_nft.move

```diff
@@ -0,0 +1,114 @@
+// Auto-generated by Move decompiler
+
+module 370092bfce3c986971f00a4f54bc2a6cc98dfcdacc48cd48626c6bc66d685f71::cat_nft;
+
+// -- structs -- 
+
+public struct CatNFT
+has key, store {
```

Comment:
I know the autoformatter will all take care of this eventually so it doesn't matter, but this line break makes me hurt lol